app/error-handlers.js

```diff
@@ -1,10 +1,29 @@
+const { response } = require('./app');
+
 const badPath = (request, response, next) => {
   response.status(404).send({ msg: 'Not Found' });
 };
 
+const customErrorHandler = (err, request, response, next) => {
+  if (err.status && err.msg) {
+    response.status(err.status).send({ msg: err.msg });
+  } else next(err);
+};
+
+const sqlInputErrors = (err, request, response, next) => {
+  if (err.code === '22P02') {
+    response.status(400).send({ msg: 'Bad Request' });
+  } else next(err);
+};
+
 const handle500errors = (err, request, response, next) => {
   console.log(err);
```

Comment:
Remove console logs please

Reply:
Put them in when developing, but remove when committing code that will be in the main